docs/source/queries/batch.md

```diff
@@ -1,8 +1,8 @@
 # Batch statement
 
-A batch statement allows to run many queries at once.\
-These queries can be [simple queries](simple.md) or [prepared queries](prepared.md).\
-Only queries like `INSERT` or `UPDATE` can be in a batch.
+A batch statement allows to run many statements at once.\
+These statements can be [simple](simple.md) or [prepared](prepared.md).\
+Only `INSERT`, `UPDATE` and `DELETE` statements are allowed.
```

Comment:
I'm not convinced about _"running statements"_. _Statement,_ as I understand it, is a passive object containing a string contents ("the statement"), whereas `running` implies some kind of a process, an action. A _query_, on the other hand, is a process of asking the DB for some information. To sum up, "_running_" goes better with some action word.
Another word denoting a related process is _request_.  _Request_ is wider than _query_ in that it allows both for `SELECT`s (which are essentially questions to the DB) and mutating statements.
I suggest replacing _"running statements"_ as well as _"running queries"_ with _"running requests"_ or _"issuing requests"_.

Another combination that I find acceptable is _"executing statements"_, but _"requests"_ are IMO more suitable.

Reply:
> I'm not convinced about _"running statements"_. _Statement,_ as I understand it, is a passive object containing a string contents ("the statement"), whereas `running` implies some kind of a process, an action. A _query_, on the other hand, is a process of asking the DB for some information. To sum up, "_running_" goes better with some action word.

Yes, "running statements" sounds a bit strange.

> Another word denoting a related process is _request_. _Request_ is wider than _query_ in that it allows both for `SELECT`s (which are essentially questions to the DB) and mutating statements. I suggest replacing _"running statements"_ as well as _"running queries"_ with _"running requests"_ or _"issuing requests"_.

No, "request" is too broad in my opinion. "Statement" is more specific, and maybe not specific enough as "statement" can mean both "INSERT"/"UPDATE"/"DELETE" which can be sent in batches as well as "SELECT"s which cannot.

> Another combination that I find acceptable is _"executing statements"_, but _"requests"_ are IMO more suitable.

"Statement execution" is a phrase that appears in our docs (https://docs.scylladb.com/stable/cql/dml.html#batch-statement), so I'll change to "executing statements".